src/firmware_new/proto/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@rules_proto//proto:defs.bzl", "proto_library", "proto_lang_toolchain")
+load("@com_google_protobuf//:protobuf.bzl", "proto_gen")
+load(":nanopb.bzl", "c_proto_library")
+
+package(default_visibility = ["//visibility:public"])
+
+# add more proto here
+proto_files = [
+	"control.proto",
+]
+
+# proto files
+proto_library(
+    name = "fw_proto",
+    srcs = proto_files,
+    visibility = ["//visibility:private"]
+)
+
+# c library (TODO since this a macro we lack context to use :fw_proto) check w/ Gareth
```

Comment:
I don't want to block on this, but can you please file an issue for me and I can rewrite this to be a full-blown bazel rule (so it can use fields in the same sorta way we would with something like `cc_library`, etc.)

Reply:
yeah it lacks Bazel "context" (TODO edit this w/ ticket number)